Shared/UI/Face Detection/FaceImageCaptureViewController.swift

```diff
@@ -277,8 +319,9 @@ extension FaceImageCaptureViewController: AVCaptureVideoDataOutputSampleBufferDe
                             &pixelBuffer)
 
         let context = CIContext()
-        context.render(self.currentCIImage!, to: pixelBuffer!)
-
+        let transform = CGAffineTransform(translationX: 0, y: -240)
+        let adjustedImage = self.currentCIImage!.transformed(by: transform)
+        context.render(adjustedImage, to: pixelBuffer!)
```

Comment:
Shifting the video frames down so the face is centered. -240 is just a magic number to get it working. Ideally we can determine this based on the face position.